proto/sdk/sdk.proto

```diff
@@ -129,13 +129,20 @@ message GameServer {
 
     message Spec {
         Health health = 1;
+        SdkServer sdkServer = 2;
 
         message Health {
             bool disabled = 1;
             int32 period_seconds = 2;
             int32 failure_threshold = 3;
             int32 initial_delay_seconds = 4;
         }
+
+        message SdkServer {
```

Comment:
Not sure I'm understanding why the proto change? I don't see any major reason to pass log level information to the SDK user?

Reply:
This `SdkServer` and `SdkServer.logLevel` are not used in following section in my PR.
https://github.com/googleforgames/agones/pull/2580/files#diff-a02544ece56edb873e82b56146e058a2cc36f010391d705b826899d17db824ffR666-R679

But these are used in test codes which I added.
https://github.com/googleforgames/agones/pull/2580/files#diff-a11e2ffffc7affd5f7777c02cba93464d213f1389e2a5451edf8ad317deec07eR114-R137

In the current proto file, `SdkServer` is not defined.
```
	fixture := &agonesv1.GameServer{
		ObjectMeta: metav1.ObjectMeta{Name: "stuff"},
		Spec: agonesv1.GameServerSpec{
			SdkServer: agonesv1.SdkServer{LogLevel: "Debug"},
		},
	}
```
Without `SdkServer` definition, the above code returns the nil spec GameServer.

Reply:
🤔 you have access to the private variables within LocalSDK, so you could check the `*gs` value directly, rather than loading it this way - then you don't need the proto change.

Reply:
Thank you for your comment. 
As you said, there was no need to change the proto.
It could confirm just by checking the private variables in LocalSDK.

I reverted proto changes.